pxr/usd/lib/usd/stage.h

```diff
@@ -700,6 +700,33 @@ class UsdStage : public TfRefBase, public TfWeakBase {
     USD_API
     UsdPrim GetPrimAtPath(const SdfPath &path) const;
 
+    /// Return the UsdObject at \p path, or an invalid UsdObject if none exists.
+    ///
+    /// If \p path indicates a prim beneath an instance, returns an instance
```

Comment:
Can we add to this doc to mention properties and instance proxies as well? Something like,

```
If \p path indicates a property beneath a child of an instance, returns a property
whose parent prim is an instance proxy prim.
```

Reply:
Thanks for the notes, I'll get an updated branch pushed up this weekend.

Reply:
Sorry for the delay @sunyab, updated